### PR TITLE
Score star rating

### PR DESCRIPTION
**Pull request by j-or**

### CODE REVIEW

File: packages/core/src/common/util/converters/converters.ts [hunk 1]
```diff
@@ -67,19 +67,38 @@ export const calculateData = (
...
         yi: run
           .filter(it => enabledScoreNames.includes(it.name))
-          .map(it => it.value)
-          .map(Number),
+          .map(it => invertScore(arr, it)),
       })
     )
 }
+
+/** for a given score, returns ('max score with the same name' - score)
```
Comment (j-or) [inline packages/core/src/common/util/converters/converters.ts:83]: @langdal Is this what we want?

Comment (j-or) [reply packages/core/src/common/util/converters/converters.ts]: Inversion removed for now

File: packages/ui/src/features/core/editable-table/editable-table-expanded-row.tsx [hunk 2]
```diff
@@ -92,18 +93,19 @@ export const EditableTableExpandedRow = ({
                     ))}
                   </TableRow>
                   <TableRow>
-                    {editedRow.dataPoints.map((d, i) => (
+                    {editedRow.dataPoints.map((d, i, arr) => (
                       <EditableTableCell
                         key={'expandedvalues' + i}
                         value={d.value}
+                        isRatingInput={
+                          arr.findIndex(d => d.name.includes('score')) === i
```
Comment (langdal) [inline packages/ui/src/features/core/editable-table/editable-table-expanded-row.tsx:101]: We should identify the score cell either by adding a "data type" to the EditDataType, e.g., type = numeric|string|star|options or by adding a prop at the top level EditableTable(ratingColumns: string[])

**Review by langdal — APPROVED** [review]

File: packages/ui/src/features/core/editable-table/editable-table-expanded-row.tsx (hunk 2 above)
Comment (j-or) [reply packages/ui/src/features/core/editable-table/editable-table-expanded-row.tsx]: Type added

**Pull request merged**